build/cmake/FindRe2.cmake

```diff
@@ -2,15 +2,27 @@ include(ExternalProject)
 
 SET(RE2_VERSION "2018-10-01")
 
-ExternalProject_Add(re2
-    DOWNLOAD_COMMAND git clone --quiet --depth 1 --branch ${RE2_VERSION} --config advice.detachedHead=false https://github.com/google/re2.git
-    TIMEOUT 5
-    INSTALL_COMMAND ""
-    CONFIGURE_COMMAND ""
-    BUILD_IN_SOURCE TRUE
-    BUILD_COMMAND ${CMAKE_COMMAND} -E env ARFLAGS=-r\ -s\ -c CXXFLAGS=-O3\ -g\ -fPIC\ -D_GLIBCXX_USE_CXX11_ABI=0 make -j
-    BUILD_BYPRODUCTS ${CMAKE_CURRENT_BINARY_DIR}/re2-prefix/src/re2/obj/libre2.a
-)
+if (ISMACOS)
```

Comment:
same comment about extracting args if it works

Reply:
yeah, in this case is trickier, we have an `LDFLAGS` in one, and not in another, and I don't want to overwrite it.

Reply:
Yeah no worries 👍